### DISCUSSION TITLE
docker-compose and mysql container

### DISCUSSION THREAD

**Discussion by samo9789** [abiosoft/colima#249] [Q&A] [answered]
Hi, 

I am posting hoping for some help. I am running Colima on a Mac mini (M1)
I have installed Colima and docker with brew as documented. 
I laso installed portainer with

`docker run -d -p 9090:9000 --restart always -v /var/run/docker.sock:/var/run/docker.sock -v /opt/portainer:/data portainer/portainer`

I am trying to run mysql in a container with docker-compose up -d with a shared volume such as

volumes:
      - ./datadir:/var/lib/mysql

I am getting permissions error on /var/lib/mysql

Can someone point me to what I am missing ? 

Thank you

**Comment by jklewa** [comment, marked as answer]
This is issue #83 and you can follow https://github.com/abiosoft/colima/pull/251 to see when the fix will be available.
